### PR TITLE
Marker features

### PR DESCRIPTION
**Pull request by DeonBest**
Added marker and photo viewing features

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -47,4 +47,5 @@ dependencies {
     testImplementation 'junit:junit:4.+'
     androidTestImplementation 'androidx.test.ext:junit:1.1.2'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
+    compileOnly 'com.google.android.wearable:wearable:2.8.1'
```
Comment (dmccardle) [inline app/build.gradle:49]: What does this do?

File: app/src/main/res/values-round/strings.xml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="hello_world">Hello Round World!</string>
+</resources>
```
Comment (dmccardle) [inline app/src/main/res/values-round/strings.xml]: Is this file required?

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -13,6 +13,8 @@
     <string name="text_description">Description</string>
     <string name="text_album">Album</string>
     <string name="text_album_title">Album Title</string>
+    <string name="title_activity_photo_view">PhotoViewActivity</string>
+    <string name="add_to_album">Add to album</string>
```
Comment (dmccardle) [inline app/src/main/res/values/strings.xml:19]: It looks like this may have been added back in during the rebase and not actually used anywhere? can delete if that's the case

File: app/src/main/res/values-round/strings.xml (hunk 2 above)
Comment (DeonBest) [reply app/src/main/res/values-round/strings.xml]: Both of those are from when I accidently clicked the smartwatch activity instead of the default one.

Comment (DeonBest) [reply app/src/main/res/values-round/strings.xml]: They're not needed.
